ajax/libs/autonumeric/package.json

```diff
@@ -1,37 +1,51 @@
 {
   "name": "autonumeric",
-  "version": "1.9.46",
-  "description": "autoNumeric is a jQuery plugin that automatically formats currency (money) and numbers as you type on form inputs. It supports most International numeric formats and currency signs including those used in Europe, North and South America, Asia and India's' lakhs",
-  "filename": "autoNumeric.min.js",
+  "version": "4.0.0-beta.2",
```

Comment:
@AlexandreBonneau Actually, we don't need to modify version field because it will automatically update by auto-updater in cdnjs.